model.json

```diff
@@ -0,0 +1,61 @@
+{
+    "interactionModel": {
+        "languageModel": {
+            "invocationName": "Activity Suggestion",
+            "intents": [
+                {
+                    "name": "AMAZON.CancelIntent",
+                    "samples": [
+                        "exit",
+                        "quit",
+                        "cancel"
```

Comment:
Built-in intents don't need samples unless you're adding something that's not covered by Amazon. These three definitely trigger cancel.